spec/js/ko-components/view-models/orifice-calculator-spec.coffee

```diff
@@ -117,3 +117,44 @@ describe 'orifice-calculator-viewmodel-spec', ->
 
     it 'should have None and Zf for compressibility correction', ->
       expect(viewModel.compressibilityCorrection()).toEqual _.values OPL.OrificeCalculator.Config.Dictionaries.CompressibilityCorrection
+
+  describe 'betaRatio', ->
+    it 'should return beta ratio', ->
+      viewModel.selectedPipeID OPL.OrificeCalculator.Config.Dictionaries.PipeID.twoZeroInch.value
+      viewModel.orificeBoreDiameter 0.97
+      expect(viewModel.betaRatio()).toEqual 0.47
+
+  describe 'velocityOfApproach', ->
+    it 'should return velocity of approach', ->
+      viewModel.orificeBoreDiameter 0.97
+      viewModel.selectedPipeID OPL.OrificeCalculator.Config.Dictionaries.PipeID.oneNineInch.value
+      expect(viewModel.velocityOfApproach()).toEqual 1.04
+      viewModel.orificeBoreDiameter 0.776
+      viewModel.selectedPipeID OPL.OrificeCalculator.Config.Dictionaries.PipeID.oneNineInch.value
+      expect(viewModel.velocityOfApproach()).toEqual 1.02
+
+  describe 'flowRate', ->
+    it 'should return the flow rate', ->
+      viewModel.orificeBoreDiameter 0.97
+      viewModel.selectedPipeID  OPL.OrificeCalculator.Config.Dictionaries.PipeID.oneNineInch.value
+      viewModel.operatingPressure 900
+      viewModel.compressibilityCorrectionValue 1
+      viewModel.differentialPressure 30
+      viewModel.baseSpecificGravity 1
+      viewModel.operatingTemperature 60
+      expect(viewModel.flowRate()).toEqual 543.783
+
+      viewModel.orificeBoreDiameter 0.776
+      viewModel.selectedPipeID OPL.OrificeCalculator.Config.Dictionaries.PipeID.oneNineInch.value
+      viewModel.operatingPressure 900
+      viewModel.compressibilityCorrectionValue 1
+      viewModel.differentialPressure 30
+      viewModel.baseSpecificGravity 1
+      viewModel.operatingTemperature 60
+      expect(viewModel.flowRate()).toEqual 341.328
+
+  describe 'Math.ceil10', ->
```

Comment:
Why are you testing that javascript works as supposed? Did you invent this method? No.
So no need to test